syndeo-template/src/test/java/com/google/cloud/syndeo/perf/AnyToAnySyndeoTestLT.java

```diff
@@ -0,0 +1,505 @@
+                psrm.createSubscription(rpath, tname, "subscr-wr-" + TEST_ID);
+                return SinkAndSourceConfigs.create(
+                    buildJsonConfig(
+                        "beam:schematransform:org.apache.beam:pubsublite_write:v1",
+                        configToMap(
+                            PubsubLiteWriteSchemaTransformProvider
+                                .PubsubLiteWriteSchemaTransformConfiguration.builder()
+                                .setFormat("AVRO")
+                                .setLocation(rpath.location().value())
+                                .setProject(rpath.project().toString())
+                                .setTopicName(tname.value())
+                                .build())),
+                    null);
+              }),
+          "syndeo:schematransform:com.google.cloud:bigtable_write:v1",
+          TransformProvider.create(
+              wrap(() -> DefaultBigtableResourceManager.builder(TEST_ID, PROJECT).build()),
+              (ResourceManager rm) -> {
+                assert rm instanceof BigtableResourceManager;
+                BigtableResourceManager btrm = (BigtableResourceManager) rm;
+                btrm.createInstance(
+                    Collections.singletonList(
+                        BigtableResourceManagerCluster.create(
+                            "cluster-" + TEST_ID.substring(0, 15),
+                            REGION + ZONE_SUFFIX,
+                            10,
+                            StorageType.SSD)));
+                String tableName = ("table-" + TEST_ID).substring(0, 29);
+                btrm.createTable(
+                    tableName, SyndeoLoadTestUtils.NESTED_TABLE_SCHEMA.getFieldNames()); // TODO
+                return SinkAndSourceConfigs.create(
+                    buildJsonConfig(
+                        "syndeo:schematransform:com.google.cloud:bigtable_write:v1",
+                        configToMap(
+                            BigTableWriteSchemaTransformConfiguration.builder()
+                                .setInstanceId(btrm.getInstanceId())
+                                .setTableId(tableName)
+                                .setProjectId(PROJECT)
+                                .setKeyColumns(List.of("commit")) // TODO(pabloem): Figure this out
+                                .build())),
+                    null);
+              }));
+
+  static Map<String, Object> dataGeneratorConfiguration() {
+    return Map.of(
+        "urn",
+        "syndeo_test:schematransform:com.google.cloud:generate_data:v1",
+        "configurationParameters",
+        Map.of(
+            "numRows",
+            CONFIGURATIONS.get(CONFIGURATION).getNumRows(),
+            "runtimeSeconds",
+            CONFIGURATIONS.get(CONFIGURATION).getRuntime().getStandardSeconds()));
+  }
+
+  @BeforeClass
+  public void setUpSourceAndSink() {}
+
+  @Test
+  public void testSelectedSinkAndSelectedSource() throws JsonProcessingException {
```

Comment:
1. suggest change name to testRadmonlySelectedSourceandSink.
2. add another test to test with all possible source-sink combinations with two for loops. This test is convenient for regression rest.

Reply:
Can I add those changes as a follow-up to this one this week?

Reply:
👍